docs/Changelog.md

```diff
@@ -15023,6 +15023,22 @@ This version of the operator has been available since version 13 of the default
   For example, a 64-bit float 3.1415926459 may be round to a 32-bit float 3.141592. Similarly, converting
   an integer 36 to Boolean may produce 1 because we truncate bits which can't be stored in the targeted type.
 
+  In more detail, the conversion among numerical types should follow these rules:
+
+  * Casting from float to:
+    * float: +/- infinity if OOR (out of range).
+    * int, uint: undefined if OOR.
+    * bool: +/- 0.0 to False; all else to True.
+  * Casting from int, uint to:
+    * float: +/- infinity if OOR. (+ infinity in the case of uint)
+    * int, uint: when OOR, discard higher bits and reinterpret with respect to two's complement representation. For
```

Comment:
Thanks for the change. Minor nit: may be add "for signed types" after "two's complement representation", since it is not meant for unsigned.

Reply:
Updated.